x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/single_page_layout/components/post_install_cloud_formation_modal.tsx

```diff
@@ -41,9 +45,19 @@ export const PostInstallCloudFormationModal: React.FunctionComponent<{
 
   const cloudFormationProps = getCloudFormationPropsFromPackagePolicy(packagePolicy);
 
+  const fleetServerHostId = agentPolicy?.fleet_server_host_id;
+  const { data } = useGetFleetServerHosts();
+
+  const fleetServerHosts = useMemo(() => data?.items ?? [], [data?.items]);
```

Comment:
its a nit pick but i dont think you need a memo here, using memo is not free and unless it saves relatively serious rendering or calculations we should probably use it sparingly